### DISCUSSION TITLE
Set a default config path for onboarding PRs

### DISCUSSION THREAD

**Discussion by binyamin** [renovatebot/renovate#12809] [Request Help] [answered]
Currently, my onboarding PRs put the `renovate.json` in the root directory. I prefer it in the `.github` folder, and I always need to change the PR manually. How do make that the default behavior? I have an organization-level config file in [binyamin/.github](https://github.com/binyamin/.github).

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/self-hosted-configuration/#onboardingconfigfilename

This is only configurable on self-hosted renovate.

**Reply by rarkins** [reply]
We can maybe get that to work with the hosted app but it would need the feature added
